packages/solutions/app-tools/src/plugins/analyze/getHtmlTemplate.ts

```diff
@@ -43,15 +39,49 @@ const findPartials = (
   return null;
 };
 
+export const getModifyHtmlPartials = (
+  partials: Record<keyof HtmlPartials, string[]>,
+) => {
+  const append = (type: keyof HtmlPartials, ...script: string[]) => {
+    script.forEach(item => {
+      partials[type].push(item);
+    });
+  };
+  const prepend = (type: keyof HtmlPartials, ...script: string[]) => {
+    script.forEach(item => {
+      partials[type].unshift(item);
+    });
+  };
+  return {
+    top: {
+      append: (...script: string[]) => append(PartialPosition.TOP, ...script),
+      prepend: (...script: string[]) => prepend(PartialPosition.TOP, ...script),
```

Comment:
So, `append` and `prepend` is the new API for modifyHtmlPartials hook ?

Reply:
yes